public/app/plugins/datasource/influxdb/components/ConfigEditor.tsx

```diff
@@ -91,6 +91,7 @@ export class ConfigEditor extends PureComponent<Props, State> {
             <InlineFormLabel className="width-10">Organization</InlineFormLabel>
             <div className="width-10">
               <Input
+                aria-label="Organization"
```

Comment:
Haven't tested but I think we could fix these issues by using new `Input` with `id` prop inside `InlineInput` components, not a huge fan of adding `aria-label` to input components as there are better ways to make them accessible imo.

Reply:
Oh, that's a good point ( I missed we could do that for Inputs 😅 ), I am not familiar with the `InlineInput` component,  but rethinking this implementation, is it a good alternative to do something like this as well?

```				
					<InlineFormLabel className="width-10" htmlFor="organizationInput">Organization</InlineFormLabel>
            <div className="width-10">
              <Input
               id="organizationInput"```

Reply:
Sorry for confusion, I meant `InlineField` component, not `InlineInput` (which we do not have) 🙈  But it should work the same way as `Field` component and we'd be able to do smth like: 
```
<InlineField label={'Organization'}>
  <Input id={'organization'} ... />
</InlineField>
```
In this case the label's `htmlFor` attribute will automatically point to the input inside of it, making it accessible.

Reply:
@Clarity-89 sorry, forgot the query-editor file, going to update that too.

Reply:
@Clarity-89 ok, all the changes in, i hope this is better now 😄